text/0000-fabric-private-chaincode-lite.md

```diff
@@ -218,8 +218,6 @@ The [Roll-back Protection Extension](#rollback-protection-extension) to FPC Lite
 
 ### Use case: Privacy-enhanced Federated Learning on FPC Lite
 
-***TODO: @Jeb can you please make pass on below and replace this section as appropriate with info from HBP, UMBC or alike***
```

Comment:
I would have left that for now until we talk tomorrow. Jeb probably doesn't have the cycles but if he would have, it would be good to add some linkage to HBP

Reply:
sounds good to me

Reply:
So you are reverting? (i don't think i can do that, e.g., either as a clickable suggestion or part of the merge?)